docs/application-connector/docs/050-gs-calling-registered-service-from-kyma.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Calling registered service from within Kyma
+type: Getting Started
+---
+
+This guide shows how to call the registered service from within Kyma using a simple lambda function.
```

Comment:
This guide shows how to call a registered external service from Kyma using a simple lambda function.